index.js

```diff
@@ -17,11 +19,19 @@ module.exports = function (source) {
 		return;
 	}
 
-	var compiler = new KaitaiStructCompiler();
-	var kaitaiImport = 'var KaitaiStream = require("kaitai-struct/KaitaiStream")\n';
+	const context = this.context;
```

Comment:
Please add a comment on the top of the `module.exports` function definition stating that `this` refers to https://webpack.js.org/api/loaders/#the-loader-context. I'm not familiar with Webpack and this took me a while to find. Given that `module.exports` is a native Node.js feature, I assumed that `this` in it will also have some specific Node.js-defined behavior, but all Google searches like `node.js module.exports this context` were of course doomed to fail.

Reply:
Good point, added a comment including the above link, for easy reference.